src/ii_collections/n19Sum.kt

```diff
@@ -7,5 +7,5 @@ fun example6() {
 fun Customer.getTotalOrderPrice(): Double {
     // Return the sum of prices of all products that a customer has ordered.
     // Note: a customer may order the same product for several times.
-    return orders.flatMap { it.products }.map { it.price }.sum()
+    return orders.flatMap { it.products }.sumByDouble { it.price }
```

Comment:
Ah I see! Sorry, I misunderstood your role :)
I just got really excited about potentially contributing something to the community here.